examples/19-EnsembleModel-examples.ipynb

```diff
@@ -0,0 +1,868 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #22.    from darts.utils.timeseries_generation import linear_timeseries
</pre><p>this line and the one above can be put together</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/unit8co/darts/pull/1815/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>